tests/torch/data/reference_graphs/quantized/asymmetric/densenet121.dot

```diff
@@ -726,9 +726,9 @@ strict digraph  {
 "724 DenseNet/Sequential[dense4]/Bottleneck[15]/cat_0" [id=724, type=cat];
 "725 DenseNet/NNCFBatchNorm2d[bn]/batch_norm_0" [id=725, type=batch_norm];
 "726 DenseNet/relu_0" [id=726, type=relu];
-"727 DenseNet/AsymmetricQuantizer/asymmetric_quantize_0" [id=727, type=asymmetric_quantize];
+"727 DenseNet/NNCFInterface[_nncf]/ModuleDict[external_quantizers]/AsymmetricQuantizer[DenseNet/relu_0|OUTPUT]/asymmetric_quantize_0" [id=727, type=asymmetric_quantize];
```

Comment:
Note that now the label of activation quantizers is more explicitly stated. The rest of the changes to the reference graphs are all because of this change - no structural changes in any except for the Inceptionv3 and SSD due to reasons outlined above.